client/components/button/docs/example.jsx

```diff
@@ -23,19 +25,38 @@ var Buttons = React.createClass( {
 	},
 
 	render: function() {
-		var toggleButtonsText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		var toggleText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		return config.isEnabled( 'devdocs/usage-stats' )
+			? this.renderNewStructure( toggleText )
```

Comment:
What do “new structure” and “structure” mean? Can we maybe think of a more semantic name?